test/L2/L2HodlerdropRedistribution.t.sol

```diff
@@ -157,145 +144,136 @@ contract L2AirdropV2Test is Test {
+        // Merkle root is not set so hodlerdrop has not started yet
+        vm.expectRevert("L2HodlerdropRedistribution: hodlerdrop has not started yet");
+        l2HodlerdropRedistribution.sendLSKToEcosystemWallet();
     }
 
     function test_SendLSKToEcosystemFundWallet_AirdropV2PeriodNotOver() public {
-        // proceed time to HODLER_AIRDROPV2_DURATION so that airdrop period is not over
-        vm.warp(block.timestamp + l2AirdropV2.HODLER_AIRDROPV2_DURATION() * 1 days);
+        // proceed time to HODLERDROP_REDISTRIBUTION_DURATION so that hodlerdrop period is not over
+        vm.warp(block.timestamp + l2HodlerdropRedistribution.HODLERDROP_REDISTRIBUTION_DURATION() * 1 days);
 
-        vm.expectRevert("L2AirdropV2: airdrop is not over yet");
-        l2AirdropV2.sendLSKToEcosystemWallet();
+        vm.expectRevert("L2HodlerdropRedistribution: hodlerdrop is not over yet");
+        l2HodlerdropRedistribution.sendLSKToEcosystemWallet();
     }
 
     function test_SendLSKToEcosystemFundWallet_OnlyOwner() public {
         // alice tries to send all L2LiskToken to Ecosystem Fund wallet
         vm.prank(alice);
         vm.expectRevert(abi.encodeWithSelector(Ownable.OwnableUnauthorizedAccount.selector, alice));
-        l2AirdropV2.sendLSKToEcosystemWallet();
+        l2HodlerdropRedistribution.sendLSKToEcosystemWallet();
     }
 
     function aliceSatifiesStakingTier1() internal {
         // alice stakes 30 and 50 L2LiskToken for minimum and minumum plus 1 days respectively in two positions
         vm.startPrank(alice);
-        l2Staking.lockAmount(alice, 30 * 10 ** 18, l2AirdropV2.MIN_STAKING_DURATION_TIER_1());
-        l2Staking.lockAmount(alice, 50 * 10 ** 18, l2AirdropV2.MIN_STAKING_DURATION_TIER_1() + 1);
+        l2Staking.lockAmount(alice, 30 * 10 ** 18, l2HodlerdropRedistribution.MIN_STAKING_DURATION_TIER_1());
+        l2Staking.lockAmount(alice, 50 * 10 ** 18, l2HodlerdropRedistribution.MIN_STAKING_DURATION_TIER_1() + 1);
         vm.stopPrank();
 
-        // maximum staking tier 1 airdrop amount for alice is 80 L2LiskToken
-        assertEq(l2AirdropV2.satisfiesStakingTier1(alice, 80 * 10 ** 18), true);
+        // maximum staking tier 1 hodlerdrop amount for alice is 80 L2LiskToken
+        assertEq(l2HodlerdropRedistribution.satisfiesStakingTier1(alice, 80 * 10 ** 18), true);
 
         // check that bigger amount than 80 L2LiskToken does not satisfy staking tier 1
-        assertEq(l2AirdropV2.satisfiesStakingTier1(alice, (80 * 10 ** 18) + 1), false);
+        assertEq(l2HodlerdropRedistribution.satisfiesStakingTier1(alice, (80 * 10 ** 18) + 1), false);
     }
 
     function test_SatisfiesStakingTier1() public {
         // check that alice satisfies staking tier 1
         aliceSatifiesStakingTier1();
     }
 
+    function test_charlieSatifiesStakingTier1With400LockingPositions() public {
+        vm.startPrank(charlie);
+        // charlie stakes 1 L2LiskToken for minimum days in 400 positions
```

Comment:
I think the initial intention to test such a large number was to ensure that a claim transaction doesn't run out of gas, right?
Therefore, it would be good to have a test that calls the claim function for this setup and also checks that the block gas limit was not exceeded. This could be done by the `gasleft()` function. 
Moreover, since we had already the case of 307 positions in the hodlerdrop, I would tend to choose an even larger number to ensure it will work for every user.

Reply:
Increase the number of locking positions to 1000, also check the `gasleft()`.